backend/domains/template.yaml

```diff
@@ -0,0 +1,72 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  DNS configuration for self-service
+
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Resources:
+  SignInServiceHostedZone:
+    Type: AWS::Route53::HostedZone
+    Properties:
+      HostedZoneConfig:
+        Comment: Hosted zone for sign-in.service.gov.uk
+      Name: sign-in.service.gov.uk.
+
+  FromAuth:
```

Comment:
A better name for this? `SingInService(Sub)Domains` or something like that?

Reply:
👍🏻